packages/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@viamrobotics/prime-core",
-  "version": "0.0.8",
+  "version": "0.0.9",
```

Comment:
`prime-core` is already on [`0.0.9` now](https://github.com/viamrobotics/prime/blob/main/packages/core/package.json#L3) - please bump to `0.0.10`!